src/applications/gi/containers/GiBillApp.jsx

```diff
@@ -99,6 +104,7 @@ export class GiBillApp extends React.Component {
                 onViewLiveVersion={this.exitPreviewMode}
               />
             )}
+
```

Comment:
And no need to make whitespace changes unless this was done by the linter

Reply:
Not sure how that got there but I removed it